packages/framework/src/Component/Csv/CsvReader.php

```diff
@@ -4,6 +4,9 @@
 
 use Symfony\Component\Filesystem\Exception\FileNotFoundException;
 
+/**
+ * @deprecated Class is obsolete and will be removed in the next major
```

Comment:
I like when `@deprecated` contains how to resolve it - eg. "use `SplFileObject::fgetcsv()` instead". The information in upgrade notes is usefull only during upgrade.

Reply:
There is info in the method and also in upgrade notes.

Reply:
Thanks for the point. I don't see any reason why not add additional info into depreciation. 
I added the hint "use SplFileObject::fgetcsv() instead". 
Do you agree @TomasLudvik ?